### DISCUSSION TITLE
Exception: Shape type 'UserMenu' not found

### DISCUSSION THREAD

**Discussion by xtomas** [OrchardCMS/OrchardCore#9603] [Q&A] [answered]
Hi,

I'm trying to make OrchardCore Demo and try some features. But I'm not able to go show admin dashboard.

![image](https://user-images.githubusercontent.com/10938220/120463450-3798b000-c39c-11eb-95b8-0f77cd8e796c.png)

I've tried https://www.youtube.com/watch?v=bacIbCrTsng and, https://youtu.be/Gfy5SCACyL8?t=1604 and clear DotnetCore 3.1 Web App with step by [step documentation](https://docs.orchardcore.net/en/dev/docs/guides/add-admin-menu/).

Nothing helped and I always recieve shape not found exception.

What am I doing wrong?

**Comment by jptissot** [comment]
Which version are you using ?

**Reply by xtomas** [reply]
`1.0.0-rc2-13450`

**Reply by xtomas** [reply]
I used same install steps:
* using [Init-OrchardCoreSolution](https://github.com/Lombiq/Utility-Scripts/tree/dev/OrchardCore/Init-OrchardCoreSolution)
* `.\Init-OrchardCoreSolution -Name "OrchardCore.Demo" -ModuleName "OrchardCore.Module" -ThemeName "OrchardCore.Theme"`
* `cd src/OrchardCore.Demo.Web`
* `dotnet build`
* `dotnet run`
* open http://localhost:5000 with some browser
* setup site with blank site and sqllite
* go to http://localhost:5000/admin
* login
* error

It works both on Win10+WSL/Ubuntu20.04, Windows Sandbox. On both installed 

```powershell
dotnet --list-runtimes
Microsoft.AspNetCore.App 3.1.15 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
Microsoft.NETCore.App 3.1.15 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
```

```powershell
dotnet --list-sdks
3.1.409 [/usr/share/dotnet/sdk]
```

But on my Windows 10 it doesn't work. I have a lot of sdk's and runtimes, but creating, build and run steps are equal.

```powershell
dotnet --list-sdks
2.1.816 [C:\Program Files\dotnet\sdk]
3.1.402 [C:\Program Files\dotnet\sdk]
3.1.409 [C:\Program Files\dotnet\sdk]
5.0.203 [C:\Program Files\dotnet\sdk]
5.0.300 [C:\Program Files\dotnet\sdk]

dotnet --list-runtimes
Microsoft.AspNetCore.All 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
Microsoft.AspNetCore.App 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 3.1.15 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 5.0.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.NETCore.App 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 3.1.15 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 5.0.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.WindowsDesktop.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 3.1.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 5.0.6 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by deanmarcussen** [comment, marked as answer]
The version of Oc you are using does not support that feature. You need to upgrade to the preview feed if you want it.


https://docs.orchardcore.net/en/dev/docs/getting-started/preview-package-source/

**Reply by xtomas** [reply]
I've used preview packages source and then upgraded package to latest rc2 `1.0.0-rc2-16264`. Now it works. But it seems strange to me, that version 13450 works well on linux and sandbox, but on windows it must be used latest package version.

**Reply by xtomas** [reply]
@deanmarcussen thank you for preview packages source tip 👍
